src/domain/discounts/discount-form/sections/general.tsx

```diff
@@ -187,6 +186,24 @@ const General = ({ discount, isEdit = false }) => {
               }
             />
           </div>
+
+          <div className="mt-xlarge flex items-center">
+            <Controller
+              name="is_recurring"
+              render={({ onChange, value }) => {
+                return (
+                  <Checkbox
+                    label="This is a recurring discount"
+                    name="is_recurring"
+                    id="is_recurring"
+                    checked={value}
+                    onChange={(e) => onChange(e.target.checked)}
+                    className={clsx("mr-1.5")}
+                  />
+                )
+              }}
+            />
```

Comment:
Want to add a tooltip indicating that this will apply the discount to the subscription?